.github/workflows/main.yml

```diff
@@ -174,9 +181,14 @@ jobs:
         with:
           deno-version: "1.18.2"
 
-      - name: get binary and make executable from previous step for tests
+      - name: build binary and make executable
+        env:
+          TAQ_VERSION: ${{ github.ref_name }}
+          DENO_DIR: "./deno"
```

Comment:
Is DENO_DIR even needed? I don't see where it is used

Reply:
Yeah it is used by when the `deno` command executes

Reply:
Alright cool